acceptance/suites/tests/010-puppetserver-cli/subcommand/foreground.rb

```diff
@@ -0,0 +1,44 @@
+test_name "Puppetserver 'foreground' subcommand tests."
+
+# --------------------------------------------------------------------------
+# This behavior needs to be baked into Beaker but is not yet. SERVER-79 tracks
+# the need to fix this once Beaker has been modified to make the paths to these
+# commands available.
+#
+if master.is_pe?
```

Comment:
Hmmm, actually it looks like "cli" is being overloaded later in the test to refer to either the service name (via resource service) or the command name (via ... foreground).  The service name is "puppetserver" for FOSS but is "pe-puppetserver" for PE.  The command name, however, is "puppetserver" for both FOSS and PE.  We don't ever run these tests against a PE install so the difference probably isn't a big deal anyway, but would be good to have the usages of the name reflect how it "should" work.